### DISCUSSION TITLE
Make it possible to transfer bullets between magazines

### DISCUSSION THREAD

**Discussion by leonard84** [FakeFishGames/Barotrauma#8429] [Ideas and suggestions]
Currently, it is impossible to transfer ammunition from a half full magazine to another half full magazine. So you have the choice to go into a fight with 5 shots left having to frantically reload or you end up with multiple half empty magazines.

---
Update

It seems that the Rising Tide upgrade made it possible to transfer transfer bullets from one Magazine to the other, as long as they are not inside a weapon (unless you specifically drag it onto the magazine in the weapon not the weapon itself).

**Comment by GoodGuyJimmy** [comment]
By that, do you mean like, the magazines should have their own slot for bullets kinda like how the weapons and tools have? If so, I think that's a pretty cool idea, so we could craft bullets to refill already existing magazines instead of making entirely new ones. That might also introduce weapons that share the same ammo type but in different ways, for example adding a carbine that fires the same .38 rounds from the revolver, but requires a magazine.

**Reply by Dailyyy** [reply]
no, they mean like how if you drag say, a 30% o2 tank onto a 50% o2 tank, it becomes a 0% o2 tank and a 80% o2 tank

**Reply by GoodGuyJimmy** [reply]
Yeah that's also cool

**Comment by Ek-Videogames** [comment]
+1
This is a one-line XML change, and would help with better ammo rationing ( less throwing away magazines )

**Comment by mygamingaccount** [comment]
Repacking mags has been working for ages now

IMO you should be able to repack coilgun ammoboxes too by default, but at the moment you need this mod to do that:
https://steamcommunity.com/sharedfiles/filedetails/?id=2200185630
